compass/ocean/tests/utility/create_salinity_restoring/extrap_salin.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import xarray as xr
+from scipy.signal import convolve2d
```

Comment:
```suggestion
from datetime import datetime
from scipy.signal import convolve2d
```
(Needed for datestring below.)